scripts/conversion_script.py

```diff
@@ -81,6 +82,33 @@ def to_dict(self):
         }
 
 
+def get_rhel_version():
```

Comment:
and maybe rename it to 
```suggestion
def get_system_distro_version():
```

Reply:
updated, thanks!